src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -126,159 +117,157 @@ class CompanyStep extends React.Component {
-                            defaultValue={this.props.getDefaultStateForField('incorporationState')}
-                            shouldSaveDraft
-                        />
-                    </View>
-                    <CheckboxWithLabel
-                        accessibilityLabel={`${this.props.translate('companyStep.confirmCompanyIsNot')} ${this.props.translate('companyStep.listOfRestrictedBusinesses')}`}
-                        inputID="hasNoConnectionToCannabis"
-                        defaultValue={this.props.getDefaultStateForField('hasNoConnectionToCannabis', false)}
-                        LabelComponent={() => (
-                            <Text>
-                                {`${this.props.translate('companyStep.confirmCompanyIsNot')} `}
-                                <TextLink
-                                    // eslint-disable-next-line max-len
-                                    href="https://community.expensify.com/discussion/6191/list-of-restricted-businesses"
-                                >
-                                    {`${this.props.translate('companyStep.listOfRestrictedBusinesses')}.`}
-                                </TextLink>
-                            </Text>
-                        )}
-                        style={[styles.mt4]}
+                </View>
+                <View style={[styles.mt4, styles.mhn5]}>
+                    <StatePicker
+                        inputID="incorporationState"
+                        label={translate('companyStep.incorporationState')}
+                        defaultValue={getDefaultStateForField('incorporationState')}
                         shouldSaveDraft
                     />
-                </Form>
-            </ScreenWrapper>
-        );
-    }
+                </View>
+                <CheckboxWithLabel
+                    accessibilityLabel={`${translate('companyStep.confirmCompanyIsNot')} ${translate('companyStep.listOfRestrictedBusinesses')}`}
+                    inputID="hasNoConnectionToCannabis"
+                    defaultValue={getDefaultStateForField('hasNoConnectionToCannabis', false)}
+                    LabelComponent={() => (
+                        <Text>
+                            {`${translate('companyStep.confirmCompanyIsNot')} `}
+                            <TextLink
+                                // eslint-disable-next-line max-len
+                                href="https://community.expensify.com/discussion/6191/list-of-restricted-businesses"
+                            >
+                                {`${translate('companyStep.listOfRestrictedBusinesses')}.`}
+                            </TextLink>
+                        </Text>
+                    )}
+                    style={[styles.mt4]}
+                    shouldSaveDraft
+                />
+            </Form>
+        </ScreenWrapper>
+    );
 }
 
 CompanyStep.propTypes = propTypes;
```

Comment:
Oh actually can we add the `displayName` to this please?

Reply:
You're right, I missed this bit